packages/service-utils/src/cf-worker/index.ts

```diff
@@ -54,14 +57,14 @@ export async function authorizeWorker(
 
   return await authorize(authData, serviceConfig, {
     get: async (clientId: string) => serviceConfig.kvStore.get(clientId),
-    put: (clientId: string, data: TeamAndProjectResponse) =>
+    put: (clientId: string, teamAndProjectResponse: TeamAndProjectResponse) =>
       serviceConfig.ctx.waitUntil(
         serviceConfig.kvStore.put(
           clientId,
           JSON.stringify({
             updatedAt: Date.now(),
-            data,
-          }),
+            teamAndProjectResponse,
```

Comment:
This needs to be named `teamAndProjectResponse` because that's [the field being read from the cache](https://github.com/thirdweb-dev/js/blob/57c4e2faab2f97f4dc1fe798c911e6cbac9b11cb/packages/service-utils/src/core/authorize/index.ts#L70).